udata/api/oauth2.py

```diff
@@ -295,7 +306,7 @@ def revoke_token():
 def authorize(*args, **kwargs):
     if request.method == "GET":
         try:
-            grant = oauth.validate_consent_request(end_user=current_user)
+            grant = oauth.get_consent_grant(end_user=current_user)
```

Comment:
does `get_consent_grant` also `validate`?

Reply:
I believe this is a simple function renaming between [0.14.3](https://docs.authlib.org/en/v0.14.3/flask/2/api.html?highlight=validate_consent_request#authlib.integrations.flask_oauth2.AuthorizationServer.validate_consent_request) and [1.3.1](https://docs.authlib.org/en/v1.3.1/specs/rfc6749.html#authlib.oauth2.rfc6749.AuthorizationServer.get_consent_grant), as both their documentation are exactly the same.

I double checked, and indeed in the `get_consent_grant` function, there is a call to `validate_content_request`: https://github.com/lepture/authlib/blob/v1.3.1/authlib/oauth2/rfc6749/authorization_server.py#L217